young_yacks/game/rules.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
```

Comment:
We need to import `CowBreed` & `EarthMetric`

Reply:
That's the __future__.annotations thing--if CowBreed is imported, we get a circular dependency problem between models and rules and the following error:

```
ImportError: cannot import name 'CowBreed' from 'game.models' (D:\Programming\code-jam-5\young_yacks\game\models
.py)
```

__future__.annotations solves the issue by delaying the actual check and avoiding the NameError that normally appears when something isn't imported.  

Described in detail at https://stackoverflow.com/questions/33837918/type-hints-solve-circular-dependency